scripts/process_jartic_parallel.py

```
@@ -0,0 +1,320 @@
+import time
+from src.infrastructure.data_reference import ExternalDataManager
+
+
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(levelname)s - %(message)s'
+)
+logger = logging.getLogger(__name__)
+
+
+def process_prefecture_data(pref_data_tuple):
```

Ruff (0.12.2)

`process_prefecture_data` is too complex (14 > 10)

(C901)